app/routes.php

```diff
@@ -1,15 +1,14 @@
 <?php
 declare(strict_types=1);
 
+use Psr\Http\Message\RequestInterface;
+use Slim\Http\Response;
```

Comment:
not actually using these in the file?

Reply:
No but they will be used when we're just doing functional routes without controller classes - IE when I put the old route back in, they will be used again